tests/functional/jwt_helper.js

```diff
@@ -87,35 +84,52 @@ export function generateToken(
   };
 
   // Payload
-  const oPayload = {
-    tx_id: uuidv4(),
-    jti: uuidv4(),
-    iat: KJUR.jws.IntDate.get("now"),
-    exp: KJUR.jws.IntDate.get("now") + 1800,
-    user_id: userId,
-    case_id: uuidv4(),
-    ru_ref: "12346789012A",
-    response_id: responseId,
-    ru_name: "Apple",
-    trad_as: "Apple",
-    schema_name: `${schemaParts[1]}_${schemaParts[2]}`,
-    collection_exercise_sid: collectionId,
-    period_id: periodId,
-    period_str: periodStr,
-    ref_p_start_date: "2017-01-01",
-    ref_p_end_date: "2017-02-01",
-    employment_date: "2016-06-10",
-    return_by: "2017-03-01",
-    country,
-    locality,
-    town_name: townName,
-    postcode,
-    display_address: displayAddress,
-    region_code: regionCode,
-    language_code: languageCode,
-    sexual_identity: sexualIdentity,
-    account_service_url: "http://localhost:8000",
-  };
+  let payload = {};
+
+  if (version === "v2") {
+    payload = {
+      tx_id: uuidv4(),
+      jti: uuidv4(),
+      iat: KJUR.jws.IntDate.get("now"),
+      exp: KJUR.jws.IntDate.get("now") + 1800,
+      case_id: uuidv4(),
```

Comment:
The keys in both payload should be added after you created what is unique about them. That way, it avoids the duplication of shared keys.